demo_nodes_cpp/src/parameters/even_parameters_node.cpp

```diff
@@ -16,13 +16,18 @@
 #include <vector>
 
 #include "rclcpp/rclcpp.hpp"
+#include "rclcpp_components/register_node_macro.hpp"
 
+namespace demo_nodes_cpp
+{
 class EvenParameterNode : public rclcpp::Node
 {
 public:
-  EvenParameterNode()
-  : Node("even_parameters_node")
+  explicit EvenParameterNode(const rclcpp::NodeOptions options)
+  : Node("even_parameters_node", options)
   {
+    // Force flush of the stdout buffer.
+    setvbuf(stdout, NULL, _IONBF, BUFSIZ);
```

Comment:
I wouldn't recommend this withing a node. Simply because what other nodes specify might collide. This should happen on the container level instead.